app/views/projects/show.html.erb

```diff
@@ -25,12 +26,13 @@
         <tbody id="stories" data-url="<%= sort_stories_project_path(@project.id) %>" data-unlocked="<%= is_unlocked?(@project) %>">
           <% if @stories.present? %>
             <% @stories.each do | story | %>
-              <tr class="project-table__row project-table__row--story" id="<%= dom_id(story)%>" >
+              <tr class="project-table__row project-table__row--story" id="<%= dom_id(story)%>" data-status="<%= status_string(story) %>">
```

Comment:
I understand this is here for the filtering, I would remove it from this PR for now to avoid confusion